tests/test_hf_file_system.py

```diff
@@ -310,14 +340,14 @@ def test_find_root_directory_no_revision_with_incomplete_cache(self):
             repo_type="dataset",
         )
 
-        files = self.hffs.find(self.hf_path, detail=True)
+        files = copy.deepcopy(self.hffs.find(self.hf_path, detail=True))
```

Comment:
Why do we need `copy.deepcopy` here?

Reply:
In the next lines we `.pop()` the cache, which also modifies the `files` dictionary and make the test fail